.babelrc

```diff
@@ -16,5 +16,11 @@
                 "loose": true
             }
         ]
-    ]
+    ],
+    "env": {
+        "test": {
+            "presets": ["@babel/preset-env", "@babel/preset-react"],
+            "plugins": [["transform-es2015-modules-commonjs", {"loose": true}]]
+        }
+    }
```

Comment:
This is infrastructure I've put in for jest testing ES6 modules, I pulled the actual tests from this PR but having this infrastructure in place will help when we _do_ decide to obey the testing goat for JS again.  Same is true for babel.config.json.